python/lsst/ts/mtrotator/rotator_csc.py

```diff
@@ -122,6 +123,20 @@ def __init__(
         # for the next telemetry received when this timer expires
         self.next_clock_offset_task = utils.make_done_future()
 
+        # This is to keep the backward compatibility of ts_xml v20.1.0 that
+        # does not have the following commands defined in xml.
+        # TODO: Remove this after ts_xml v21.0.0.
```

Comment:
Please review the ts_xml version. Also, all TODOs require a DM ticket so please open one and mention it here.

Reply:
Updated in the commit: [3ed04af](https://github.com/lsst-ts/ts_mtrotator/pull/86/commits/3ed04af203aa5ccebcfea3464dfa1b4652aea1f6). Thanks!